osv/models.py

```diff
@@ -13,14 +13,14 @@
 # limitations under the License.
 """Datastore types."""
 
-import datetime
+from datetime import datetime
 import enum
 import logging
 import re
 import os
 
 from urllib.parse import urlparse
-from typing import List
+from typing import List, Self
```

Comment:
is `List` still used in this file?

Reply:
Replaced uses of List with list, and removed this import.